generate-templates.yaml

```diff
@@ -35,6 +35,33 @@
        - rhel7.6
       osinfoname: "{{ oslabels[0] }}"
 
+  - name: Generate desktop RHEL 7 templates
+    template:
+      src: rhel7.tpl.yaml
+      dest: "{{ playbook_dir }}/dist/templates/{{ os }}-{{ item.workload }}-{{ item.type }}-{{ item.flavor }}.yaml"
+    with_items:
+    - {type: desktop, flavor: tiny, workload: generic, memsize: "1G", cpus: 1, iothreads: False}
```

Comment:
Actually the workload generic is supposed to be split into desktop, server and generic (= server)

Reply:
There is no need to invent `type` as well.